docs/user/dashboard/drilldowns.asciidoc

```diff
@@ -1,52 +1,246 @@
 [role="xpack"]
 [[drilldowns]]
-== Use drilldowns for dashboard actions
+== Create custom dashboard actions
 
-Drilldowns, also known as custom actions, allow you to configure a
-workflow for analyzing and troubleshooting your data.
-For example, using a drilldown, you can navigate from one dashboard to another,
-taking the current time range, filters, and other parameters with you,
-so the context remains the same. You can continue your analysis from a new perspective.
+Custom dashboard actions, also known as drilldowns, allow you to create
+workflows for analyzing and troubleshooting your data. Drilldowns apply only to the panel that you created the drilldown from, and are not shared across all of the panels. Each panel can have multiple drilldowns. 
 
-[role="screenshot"]
-image::images/drilldown_on_piechart.gif[Drilldown on pie chart that navigates to another dashboard]
+{kib} supports three types drilldowns.
```

Comment:
There are two drilldown types. By **Code drilldowns** we just meant that it is possible for 3rd party developers to create their own drilldown types in addition to two existing ones. But there is no such **Code drilldown**. 
So probably information about this should be outside of the table